pipeline_browserify/compiler.py

```diff
@@ -50,29 +64,35 @@ def is_outdated(self, infile, outfile):
         # Check if we've already calculated dependencies.
         deps = getattr(self, '_deps', None)
         if not deps:
-
             # Collect dependency information.
-            command = "%s %s %s --deps %s" % (
-                getattr(settings, 'PIPELINE_BROWSERIFY_VARS', ''),
-                getattr(settings, 'PIPELINE_BROWSERIFY_BINARY', '/usr/bin/env browserify'),
-                getattr(settings, 'PIPELINE_BROWSERIFY_ARGUMENTS', ''),
+            command = (
+                pipeline_settings.get('BROWSERIFY_VARS', ''),
+                pipeline_settings.get('BROWSERIFY_BINARY', '/usr/bin/env browserify'),
+                pipeline_settings.get('BROWSERIFY_ARGUMENTS', ''),
+                "--deps",
                 self.storage.path(infile),
             )
-            dep_json = self.execute_command(command) #, cwd=dirname(infile))
-
-            # Process the output data. It's JSON, and the file's path is coded
-            # in the "file" field. We also want to save the content of each file
-            # so we can check if they're outdated, which is coded under "source".
-            deps = []
-            for dep in json.loads(dep_json.decode()):
-
-                # Is this file managed by the storage?
-                try:
-                    exists = self.storage.exists(dep['file'])
-                except SuspiciousFileOperation:
-                    exists = None
-                if exists == True or exists == False:
-                    deps.append(dep['file'])
+
+            if command[0] is '':
```

Comment:
Strings should be compared by value, not reference, so this should be `if command[0] == '':`.